release/release.go

```diff
@@ -1151,8 +1151,6 @@ func linuxRelease(v version.Version) {
 							"--edition", me,
 							"--version", mv,
 							"--packages", packagesURL,
-							"--username", os.Getenv("BARQUE_USERNAME"),
-							"--password", os.Getenv("BARQUE_PASSWORD"),
```

Comment:
can we still pass the api key via a flag? I believe the reason we were passing these as flags is that go subprocesses don't inherit the env by default

Reply:
Done.